docs/src/pages/documentation/location-and-history.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Shared Location API and History API"
+layout: "~/layouts/MarkdownLayout.astro"
+---
+
+_Last updated_: March 17, 2025
+
+## Web fragments and browser location and history
+
+One of the `UX` benefits of `Web-Fragments` as opposed to other composability patterns and technologies, is the ability of a `web-fragment`to share the browser's location and history, depending on whether they're bound or unbound.
+
+### Bound web-fragment
+
+By default, all fragments are bound. This means that internal routing in a fragment shares the browser location and it is represented in the URL bar of the browser. When navigating internally, then, the URL bar will be bound to the navigation taking place.
```

Comment:
```suggestion
By default, all Web Fragments are bound fragments. We call them bound because their `window.location` and browser history management is bound to that of the top level app.

When a navigation occurs from within a fragment, it is reflected in the browser's address bar and history. Conversely,  hitting the browser back and forward buttons will also change the internal route of the fragment.
```